### DISCUSSION TITLE
Cambio operatore

### DISCUSSION THREAD

**Discussion by ikol88** [indomus/forum#126] [HUB: Home Assistant]
Buongiorno a tutti, ho un piccolo quesito sperando che qualcuno si sia già trovato nella mia situazione.  A breve dovrò cambiare operatore e siccome ho una gran parte della mia domotica che passa via wifi ed il raspberry pi4 è collegato al modem fibra via cavo ethernet. 
Detto ciò  mi chiedevo se una volta ottenuto il nuovo modem mi basti: 
- assegnare al raspberry lo stesso IP che aveva sul modem precedente.
- Cambiare l'SSDI del WiFi mettendo il precedente con la precedente password
- aprire nel modem le stesse porte del precedente per Duck DNS.

Oppure se la procedura sarà più complessa di cosi.

Grazie a chi potrà darmi qualche delucidazione in merito.

**Comment by snakuzzo** [comment]
dal mio punto di vista...

- sì
- sì
- sì

Quando ho cambiato modem ho fatto così..

1. Mi sono segnato assegnazioni indirizzi IP in DHCP configurate sul vecchio modem
2. Mi sono segnato le varie regole di port forwarding
3. Reimpostato tutto sul nuovo modem allo stesso modo
4. Spento vecchio modem 
5. Collegato e acceso nuovo modem
6. Configurata WIFI con stessi SSID e password del modem precedente

**Reply by ikol88** [reply]
Grazie mille @snakuzzo come al solito sei sempre qua a rincuoraci nel momento del bisogno.
